doc/main.html

```diff
@@ -1076,10 +1076,11 @@ <h4>Formulae</h4>
         <h4>Special Characters</h4>
         <p>Special characters and/or character sets from languages other than base latin should be represented as thier actual unicode character, not as their HTML entity. 
```

Comment:
```suggestion
        <p>Characters should should be encoded as UTF-8-encoded Unicode codepoints, e.g.  "あ", instead of HTML entities, e.g., <code>&quot;&amp;#x3042;&quot;</code>.
```

Reply:
udpated in https://github.com/SMPTE/html-pub/pull/127/commits/0acf37cc942e5d23c753ab64c00ebe4f7ddadb95